libbeat/cmd/configtest.go

```diff
@@ -15,6 +16,7 @@ func genConfigTestCmd(name, version string, beatCreator beat.Creator) *cobra.Com
 		Run: func(cmd *cobra.Command, args []string) {
 			b, err := beat.New(name, version)
 			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error initializing beat: %s\n", err)
```

Comment:
I don't like it too much that we get these writes directly to stdout/stderr. Perhaps it would be possible to init logging with stderr on any non-run command?